google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -1608,6 +1644,22 @@ public static Builder newBuilder() {
    */
   Bucket create(BucketInfo bucketInfo, BucketTargetOption... options);
 
+  /**
+   * Create a signed upload policy for uploading objects.This method generates and signs a policy
+   * document. You can use `policy documents`_ to allow visitors to a website to upload files to
+   * Google Cloud Storage without giving them direct write access.
+   *
+   * @param bucket
+   * @param conditions
+   * @param expiration
+   * @param options
+   * @return
```

Comment:
either write some description here or remove the `@return` tag

Reply:
done